lib/core/test/unit/Cardano/Pool/DB/Properties.hs

```diff
@@ -1430,6 +1439,64 @@ prop_modSettingsReadSettings DBLayer{..} settings = do
         assertWith "Modifying settings and reading afterwards works"
             (modSettings' == settings)
 
+-- | read . put == id
+prop_putLastMetadataGCReadLastMetadataGC
+    :: DBLayer IO
+    -> POSIXTime
+    -> Property
+prop_putLastMetadataGCReadLastMetadataGC DBLayer{..} posixTime = do
+    monadicIO (setup >> prop)
+  where
+    setup = run $ atomically cleanDB
+    prop = do
+        defGCTime <- run $ atomically readLastMetadataGC
+        assertWith
+            "Reading sync time from empty db returns start of unix epoch"
+            (defGCTime == fromIntegral @Int 0)
```

Comment:
As indicated above, I think we would be better off using **`Nothing`** to represent the lack of a garbage collection.